landbosse/model/GridConnectionCost.py

```diff
@@ -63,15 +63,35 @@ def calculate_costs(self, calculate_costs_input_dict, calculate_costs_output_dic
+        # Run utility version of GridConnectionCost for project size > 15 MW:
+        if (calculate_costs_input_dict['turbine_rating_MW'] * calculate_costs_input_dict['num_turbines']) > 15:
+            if calculate_costs_input_dict['distance_to_interconnect_mi'] == 0:
+                calculate_costs_output_dict['trans_dist_usd'] = 0
             else:
-                calculate_costs_output_dict['interconnect_adder_USD'] = 0
-            calculate_costs_output_dict['trans_dist_usd'] = ((1176 * self.input_dict['interconnect_voltage_kV'] + 218257) * (calculate_costs_input_dict['distance_to_interconnect_mi'] ** (-0.1063)) * calculate_costs_input_dict['distance_to_interconnect_mi']) + calculate_costs_output_dict['interconnect_adder_USD']
+                if calculate_costs_input_dict['new_switchyard'] is True:
+                    calculate_costs_output_dict['interconnect_adder_USD'] = 18115 * self.input_dict['interconnect_voltage_kV'] + 165944
+                else:
+                    calculate_costs_output_dict['interconnect_adder_USD'] = 0
+                calculate_costs_output_dict['trans_dist_usd'] = ((1176 * self.input_dict[
+                    'interconnect_voltage_kV'] + 218257) * (calculate_costs_input_dict['distance_to_interconnect_mi'] ** (
+                    -0.1063)) * calculate_costs_input_dict['distance_to_interconnect_mi']) + calculate_costs_output_dict[
+                                                                    'interconnect_adder_USD']
+
+        # Run distributed wind version of GridConnectionCost for project size < 15 MW:
+        else:
+            # Code below is for newer version of LandBOSSE which incorporates distributed wind into the model:
+            calculate_costs_output_dict['tower_to_point_of_interconnection_usd_per_kw'] = 1736.7 * ((
+                                                                                                            calculate_costs_input_dict[
+                                                                                                                'num_turbines'] *
+                                                                                                            calculate_costs_input_dict[
+                                                                                                                'turbine_rating_MW'] * 1000) ** (
+                                                                                                        -0.272))
+            calculate_costs_output_dict['trans_dist_usd'] = calculate_costs_input_dict['num_turbines'] * \
+                                                                 calculate_costs_input_dict[
+                                                                     'turbine_rating_MW'] * 1000 * \
+                                                                 calculate_costs_output_dict[
+                                                                     'tower_to_point_of_interconnection_usd_per_kw']
```

Comment:
It is hard to read this code when lines are split like
```
dictionary[
    'key'] * X
```

Consider breaking out the values out of the keys before you use them to enhance readability:

```
num_turbines = calculate_costs_input_dict['num_turbines']
turbine_rating_MW = calculate_costs_input_dict['turbine_rating_MW']
```

Then use the simpler variable names from these assignments in the main line that calculates `calculate_costs_output_dict['trans_dist_usd']`